CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ BUG FIXES:
 * Connect: Fix a bug with leaving around ACL tokens after a service has been deregistered. [[GH-571](https://github.com/hashicorp/consul-k8s/issues/540)]
 * CRDs: Fix ProxyDefaults and ServiceDefaults resources not syncing with Consul < 1.10.0 [[GH-1023](https://github.com/hashicorp/consul-helm/issues/1023)]
 * Connect: Skip service registration for duplicate services only on Kubernetes. [[GH-581](https://github.com/hashicorp/consul-k8s/pull/581)]
+* Connect: Delete ACL token only for services that are deregistered. [[GH-599](https://github.com/hashicorp/consul-k8s/issues/599)]
```

Comment:
Do we want to combine this in any way with the changelog entry in L10? Maybe one changelog entry that could link to issue 571 and issue 599 that says "Delete ACL token for service instances that are deregistered. Note that this will not clean up leftover ACL tokens"

Reply:
yep, good idea!